jekyll/_cci2/pipelines.adoc

```diff
@@ -0,0 +1,146 @@
+
+TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a trigger is already set up.
+
+Triggers are rules that define which configuration source should be run on a given event. A project can have one or more triggers, which are associated with events from a source of change. Triggers include, but are not limited to, a push to a branch on a version control system (VCS), for example, a GitHub repository. A trigger determines an event (or events) along with a configuration source to define a pipeline. A trigger can have only one configuration source but a configuration source can have multiple triggers.
+
+A trigger kicks off a pipeline to run the workflows defined by your connected configuration source.
+
+image::{{site.baseurl}}/assets/img/docs/project-settings-triggers.png[Triggers]
+
+==== GitHub App project triggers
+
+When the CircleCI GitHub App is installed for your organization, GitHub starts to send events for the repositories you have granted access to. CircleCI listens for push and tag events. When a trigger is created CircleCI has enough information to use the event data to determine if a pipeline should be triggered.
+
+==== GitLab project triggers
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a configuration source, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repository.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-edit-trigger.png[Trigger details]
+
+**Trigger filters** allow you to determine when a trigger should initiate a build based on the parameters provided by Gitlab’s webhook. CircleCI provides some common options, for example, only build on merge requests, but you can also build your own rules using the custom filter option. For example, a custom filter would allow you to only build on a specific branch or user.
+
+image::{{site.baseurl}}/assets/img/docs/gl-preview/gitlab-preview-project-settings-customize-triggers.png[Trigger details]
+
+==== Inbound webhook triggers
+
+NOTE: Selecting an inbound webhook as a pipeline trigger is currently in **open preview**. This feature is only available for accounts that are integrated through the CircleCI GitHub App. To find out which GitHub account type you have, see the xref:github-apps-integration#[GitHub App integration] page.
+
+Setting up an inbound webhook trigger allows you to trigger pipelines from external services. Any service that can make cURL requests can be set up in this way to trigger a pipeline. For more information, see the xref:triggers-overview#trigger-a-pipeline-from-an-inbound-webhook[Triggers overview] page.
```

Comment:
There is a disclaimer we could make here. For webhooks we don't actually ask users to "choose configuration to run". We just ask them for a path and generate a configuration source for them, in the first repo associated with a Github trigger. 
It's messy and the result of a shortcut. 

I don't actually know if we want to call it out or let users figure this out, what do you think?

Reply:
<img width="667" alt="Screenshot 2024-05-02 at 22 26 39" src="https://github.com/circleci/circleci-docs/assets/19594309/bd1c82a0-4804-4275-be91-74b1a2a472b5">

Reply:
Oh yeah I think this is worth mentioning, I'll add something. Thanks!